test/controllers/api/jobs_controller_test.rb

```diff
@@ -0,0 +1,96 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.